.github/scripts/check-folders.js

```diff
@@ -45,7 +44,7 @@ async function main() {
   // Get all relevant folders that _should_ appear in workflows
   const folders = glob
     .sync('**/package.json', {
-      ignore: ['**/node_modules/**', '**/custom-engines/**'],
+      ignore: ['**/node_modules/**', '**/custom-engines/**', 'docker/_fail/**'],
```

Comment:
**Note**: `docker/_fail` tests are ignored by `check-folders.js`, which is used when linting ecosystem-tests.